docs/source/blocks/settings.md

```diff
@@ -42,13 +42,14 @@ const customBlocks = {
     // The variation level one takes precedence.
     schemaEnhancer: CustomSchemaEnhancer,
     // A block can define variations (it should include the stock, default one)
-    variations: {
-      default: { label: 'Default' },
-      custom: {
-        label: 'Custom',
-        // The variation level schema Enhancer function
-        schemaEnhancer: CustomSchemaEnhancer
-      }
+    extensions: {
+      variation: [
```

Comment:
`extensions` but `variation` ? I would call them all in plural.

Reply:
it's because it will be saved into the block data. So we'll have, for example:

```
{
    "variation": "summaryListing",
    "text": ...
}
```

The Listing block right now saves its "variation" in the "template" key: https://github.com/plone/volto/blob/f07c4e74699d7fbd202348cc1197783861eb386b/src/components/manage/Blocks/Listing/TemplateWidget.jsx#L68, though it names its "variations" as "templates" in the config.

But from my point of view, is like this: "extensions" are the available extensions. "variation" is one kind of extension, as a generic concept.